blog/zh/blog/2022/11/04/why-do-you-need-apache-apisix-when-using-amazon-lambda.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "当 Amazon Lambda 遇上 Apache APISIX 可以擦出什么火花？"
+authors:
+  - name: "陈小兰"
+    title: "Author"
+    url: "https://github.com/Hazel6869"
+    image_url: "https://github.com/Hazel6869.png"
+keywords: 
+- Amazon
+- Apache APISIX
+- Serverless
+- API 网关
+description: 本文首先介绍了什么是 Serverless，以及为什么需要 Serverless；其次，讲述了一个好的网关在 Serverless 架构下的重要性，而 APISIX 就是这样的一个网关；最后，本文重点介绍了 APISIX 中的 Serverless 类型的插件 aws-lambda，同时列举了 Apache APISIX 其它 Serverless 相关插件。
+tags: [Ecosystem]
+---
+
+> 本文首先介绍了什么是 Serverless，以及为什么需要 Serverless；其次，讲述了一个好的网关在 Serverless 架构下的重要性，而 APISIX 就是这样的一个网关；最后，本文重点介绍了 APISIX 中的 Serverless 类型的插件 `aws-lambda`，同时列举了 Apache APISIX 其它 Serverless 相关插件。
+
+<!--truncate-->
+
+> 作者[程小兰](https://github.com/Hazel6869)，API7.ai 技术工程师，Apache APISIX Contributor。
+
+## 使用 Amazon Lambda 时为什么需要 Apache APISIX?
```

Comment:
Remove this topic and upgrade remaining topics, i.e. if `###`, change to `##`, if `####`, change to `###`.

Currently, there is only 1 `##` topic in this blog, it makes the structure very weird.

```suggestion
## 使用 Amazon Lambda 时为什么需要 Apache APISIX?
```

Reply:
see the toc layout is weird.
![image](https://user-images.githubusercontent.com/36651058/224618252-dcced8ba-34d2-4ca4-8014-982983d56eea.png)